src/main/java/gov/usgs/cida/pubs/busservice/sipp/SippProcess.java

```diff
@@ -217,35 +223,30 @@ protected ProcessSummary processPublication(ProcessType processType, Information
 		//TODO		if (rtnPub.isValid() && ProcessType.SPN_PRODUCTION.equals(processType)) {
 		//TODO				updateIpdsWithDoi(rtnPub);
 		//TODO		}
-		return buildPublicationProcessSummary(rtnPub);
+		return rtnPub;
 	}
 
-	protected ProcessSummary buildPublicationProcessSummary(MpPublication mpPublication) {
-		ProcessSummary processSummary = new ProcessSummary();
-		if (mpPublication.isValid()) {
-			processSummary.setAdditions(1);
-			processSummary.setProcessingDetails("\n\tAdded to MyPubs as ProdId: " + mpPublication.getId());
-		} else {
-			processSummary.setErrors(1);
-			processSummary.setProcessingDetails("\nERROR: Failed validation.\n\t"
-					+ mpPublication.getValidationErrors().toString().replaceAll("\n", "\n\t"));
-		}
-		return processSummary;
-	}
+	protected String buildNotOkDetails(ProcessType processType, InformationProduct informationProduct, String ipNumber) {
+		String productType = informationProduct == null || informationProduct.getProductType() == null
+				? "[Not Found]" : informationProduct.getProductType();
 
-	protected String buildNotOkDetails(ProcessType processType, InformationProduct informationProduct) {
-		StringBuilder notOkDetails = new StringBuilder("\n\t").append("IPDS record not processed (\"").append(processType).append("\") -")
-				.append(" ProductType: ").append(informationProduct.getProductType());
-		if (null != informationProduct.getPublicationType()) {
-			notOkDetails.append(" Publication Type: ").append(informationProduct.getPublicationType().getText());
-		}
-		if (null != informationProduct.getPublicationSubtype()) {
-			notOkDetails.append(" PublicationSubtype: ").append(informationProduct.getPublicationSubtype().getText());
-		}
-		if (null != informationProduct.getUsgsSeriesTitle()) {
-			notOkDetails.append(" Series: ").append(informationProduct.getUsgsSeriesTitle().getText());
+		String notProcessedMess = String.format("IPDS record for IPNumber '%s' not processed", ipNumber);
+
+		StringBuilder notOkDetails = new StringBuilder("\n\t").append(notProcessedMess).append(" (\"").append(processType).append("\") -")
+				.append(" ProductType: ").append(productType);
+
+		if(informationProduct != null) {
```

Comment:
I'm surprised Codacy didn't flag this. Should there be a space between **if** and **(**?